text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,222 @@
+the target will not block forward progress of the Rust project.
+
+- Any new tier 2 target requires compiler team approval based on these
+  requirements.
+- In addition, the infrastructure team must approve the integration of the
+  target into CI, and the CI-related requirements. This review and approval
+  will typically take place in the PR adding the target to CI.
+- A tier 2 target must have value to people other than its maintainers.
+- Any new tier 2 target must have a designated team of developers on call to
+  consult on target-specific build-breaking issues, or if necessary to develop
+  target-specific language or library implementation details. (This team should
+  in almost all cases have at least 2 developers.)
+- The target must not place undue burden on Rust developers not specifically
+  concerned with that target. Rust developers may be expected to not
+  gratuitously break a tier 2 target, but are not expected to become experts in
+  every tier 2 target, and are not expected to provide target-specific
+  implementations for every tier 2 target.
+- Where possible, tier 2 targets should provide documentation for the Rust
+  community for how to build and run tests for the platform, ideally using
+  emulation.
+- The target development team should not only fix target-specific issues, but
+  should use any such issue as an opportunity to educate the Rust community
+  about portability to their target, and enhance their documentation of the
+  target.
+- The target must build reliably in CI.
+- Building the target must not take substantially longer than other targets.
+- Tier 2 targets must support building on the existing targets used for CI
+  infrastructure.
+- Tier 2 targets must not impose burden on the authors of pull requests, or
+  other developers in the community, to ensure that tests pass for the target.
+  In particular, do not post comments (automated or manual) on a PR that
+  suggests a block on the PR based on tests failing for the target. (A PR
+  author must not break the build of a tier 2 target, but need not ensure the
+  tests pass for a tier 2 target, even if notified that they fail.)
+- The target development team should regularly run the testsuite for the
+  target, and should fix any test failures in a reasonably timely fashion.
+- A tier 2 target may be demoted or removed if it no longer meets these
+  requirements. Any proposal for demotion or removal will be CCed to people who
+  have previously worked on the platform, and will be communicated widely to
+  the Rust community before being dropped from a stable release.
+- All tier 3 requirements apply.
+
+Note: some tier 2 platforms additionally have binaries built to run on them as
+a host (such as `rustc` and `cargo`). Such a platform must meet all the
+requirements above, and must additionally get the compiler and infrastructure
+team to approve the building of host tools.
```

Comment:
Do we want to mention what happens with other tools as well? For example, we build Clippy, rustfmt and such on ARM, but they've been broken for a while.

Reply:
Currently they work again.

Reply:
I added a note here that the extent of tool support may vary depending on the target.